projects/packages/forms/src/contact-form/css/file-field.css

```diff
@@ -240,6 +240,16 @@
 	height: 46px;
 }
 
+@media (prefers-reduced-motion) {
+
+	/* Remove the spinning animation for users who prefer reduced motion */
+	.jetpack-form-file-field__progress-bar {
+		animation-duration: 0.001ms !important;
+		animation-iteration-count: 1 !important;
```

Comment:
Why did you need to use `!important`? That'll make it harder for anyone to try re-style forms with CSS from theme.

Reply:
not a good reason. Will fix it.